addon-test-support/custom-assertions/tooltip.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta
+  title="Helpers & Modifiers/Modifiers/Tooltip/Assertions"
+  parameters={{
+    viewMode: 'docs',
+    previewTabs: { canvas: { hidden: true } }
+  }}
+/>
+
+# Assertions
+
+To help testing tooltip, new `assert` functions will be added. The simple way is to add the following code in `test-helper.ts`
```

Comment:
[...] new `QUnit.assert` functions can be setup by adding the following code in `test-helper.ts`